server/ntb/publish/ntb_ninjs.py

```diff
@@ -23,46 +17,94 @@ def __init__(self):
         super().__init__()
         self.format_type = "ntb_ninjs"
 
-    def format(self, article, subscriber, codes=None):
-        return super().format(article, subscriber, codes)
-
     def _transform_to_ninjs(self, article, subscriber, recursive=True):
         ninjs = super()._transform_to_ninjs(article, subscriber, recursive)
-        ninjs["version"] = str(article.get("correction_sequence", 1))
 
-        # mapping Imatrics
-        if article.get("person"):
-            ninjs["people"] = self.format_people(article)
+        ninjs["people"] = self.format_people(article)
 
-        if article.get("organisation"):
-            ninjs["organisations"] = self.format_organisation(article)
+        ninjs["organisations"] = self.format_organisations(article)
+
+        if ninjs.get("headline"):
+            ninjs["headlines"] = self.format_headlines(article)
+
+        if ninjs.get("description_text"):
+            ninjs["descriptions"] = self.format_descriptions(ninjs)
+
+        if ninjs.get("body_html"):
+            ninjs["bodies"] = self.format_bodies(ninjs)
+
+        if ninjs.get("subject"):
+            ninjs["subjects"] = self.format_subjects(ninjs)
+
+        # removable items which mapped according to Ninjs v2
+        removable_items = [
+            "body_html",
+            "description_html",
+            "description_text",
+            "charcount",
+            "wordcount",
+            "readtime",
+            "headline",
+            "subject",
+        ]
+        for item in removable_items:
+            ninjs.pop(item)
 
         return ninjs
 
     def format_people(self, article):
         peoples = []
-        persons = article["person"]
-        for item in persons:
-            peoples.append(
-                {
-                    "name": item.get("name"),
-                    "rel": item.get("description"),
-                    "qcode": item.get("qcode"),
-                    "source": item.get("source"),
-                }
-            )
-        return peoples
-
-    def format_organisation(self, article):
+        if article.get("person"):
+            persons = article["person"]
+            for item in persons:
+                altids = item.get("altids")
+                peoples.append(
+                    {
+                        "name": item.get("name"),
+                        "rel": item.get("description"),
+                        "qcode": item.get("qcode")
```

Comment:
I don't see `qcode` or `souce` in the standard, there is `literal` which could be the id.
for `uri` there is wikidata uri scheme `http://www.wikidata.org/entity/Q{id}` in case we have wikidata id,
otherwise we should do probably `imatrics:qcode`